src/pages/home/sidebar/SidebarScreen/index.js

```diff
@@ -21,20 +23,22 @@ const SidebarScreen = (props) => {
     };
 
     return (
-        <BaseSidebarScreen
-            // eslint-disable-next-line react/jsx-props-no-spreading
-            {...props}
-        >
-            <FloatingActionButtonAndPopover
-                ref={popoverModal}
-                onShowCreateMenu={createDragoverListener}
-                onHideCreateMenu={removeDragoverListener}
-            />
-        </BaseSidebarScreen>
+        <FreezeWrapper keepVisible={!props.isSmallScreenWidth}>
+            <BaseSidebarScreen
+                // eslint-disable-next-line react/jsx-props-no-spreading
+                {...props}
+            >
```

Comment:
@adamgrzybowski @WoLewicki With moving the freeze up here, I think we should make sure the FAB button opens up fine when user signs in. ie when you create a new account and user loads the app first, it should show the menu with Request money etc open

Reply:
<img width="1555" alt="image" src="https://github.com/Expensify/App/assets/36083550/f87abef7-21dc-47ce-ac0b-c5cb75a0886a">

worked :)